src/libraries/System.Memory/src/System/Buffers/Text/Base64Decoder.cs

```diff
@@ -476,10 +477,30 @@ private static unsafe void Avx2Decode(ref byte* srcBytes, ref byte* destBytes, b
             destBytes = dest;
         }
 
+        // This can be replaced once https://github.com/dotnet/runtime/issues/63331 is implemented.
+        // Note that for performance reasons this is not functionally identical on SSE3 and Arm64. On both, the input
+        // right is truncated to 4bits per lane. In addition on SSE3, the shuffle will return 0 for each input in right
+        // where the top bit is set. On Arm64 the top bit will be ignored.
+        private static Vector128<byte> SimdShuffle(Vector128<byte> left, Vector128<byte> right, Vector128<byte> maskf)
```

Comment:
Should be marked with `AggressiveInlining`?

Reply:
Will fix.